parmed/amber/parameters.py

```diff
@@ -223,7 +223,7 @@ def __init__(self, *filenames):
     #===================================================
 
     @classmethod
-    def from_leaprc(cls, fname):
+    def from_leaprc(cls, fname, ions=True, solvents=True):
```

Comment:
Oh I completely agree, I feel super uncomfortable anytime I have to add an argument to a function just to satisfy something like this - but I thought it might just be on the side of 'useful for the future' so I put it in as a conversation starter. 

The thing is: how do I do this in a conversion script? Easiest way - copy this function and make these exact modifications to it (perhaps without the arguments, just hard-coded no solvents/ions. Perhaps the more neat-seaming way - pre-modify the string to remove those - but then I'm just gonna use the parsing from this function anyway to do that, so why not just copy it all. 

Honestly, it completely doesn't matter to me which way we do it. I think it's a bit cleaner if as much code is within ParmEd and then the conversion script is just a few lines. But I know that's not so clean to you. 

Perhaps we could meet half-way and have an 'ignore' list instead? For example by passing `from_leaprc(fname, ignore=['ions', 'solvents']` you would not load any file that contains 'ions' or 'solvents' in its name string. Similarly you could pass 'nucleic' or 'amino' to it, or perhaps 'parm' or 'frcmod' if you're feeling adventurous. Clearly that's a dev functionality mostly though. What do you think?